src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,129 @@
+package duke.task;
+
+import duke.exception.DukeException;
+import duke.util.Storage;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+/**
+ * TaskList class, for in memory storage of tasks.
```

Comment:
The program structure is already very clear and well organized. but somehow, I feel that put TaskList class outside the task package can make the program structure even clearer. Do you think so?